bot/common/commands.py

```diff
@@ -77,131 +77,133 @@ async def report(ctx):
+                    ctx.author.id,
+                    hashlib.sha256("".encode()).hexdigest(),
+                    message.id,
+                    "",
+                ).steps.hash_,
+                "",
+                message.id,
+                metadata={"daos": daos},
+            ),
         )
 
-    @bot.slash_command(
-        guild_id=GUILD_IDS, description="Update your profile for a given community"
-    )
-    async def update(ctx):
-        is_guild = bool(ctx.guild)
-        if is_guild:
-            await ctx.respond("Please run this command in a DM channel")
-            return
-        else:
-            discord_rec = await get_discord_record(ctx.author.id)
-            airtable_guild_ids = discord_rec.get("fields").get("guild_id")
-            if not airtable_guild_ids:
-                embed = discord.Embed(
-                    colour=INFO_EMBED_COLOR,
-                    description="I cannot update profile because you "
-                    "have not been onboarded for any communities. Run /join in the "
-                    "discord you want to join!",
-                )
-                await ctx.response.send_message(embed=embed)
-                ctx.response.is_done()
-                return
-
-            await ctx.response.defer()
-            guild_ids = []
-            for record_id in airtable_guild_ids:
-                g = await get_guild(record_id)
-                guild_id = g.get("guild_id")
-                if guild_id:
-                    guild_ids.append(guild_id)
-            embed = discord.Embed(
-                colour=INFO_EMBED_COLOR,
-                title="Welcome",
-                description="Which community profile would you like to update?",
-            )
-            emojis = get_list_of_emojis(len(guild_ids))
-            daos = {}
-            for idx, guild_id in enumerate(guild_ids):
-                guild = await bot.fetch_guild(guild_id)
-                if not guild:
-                    continue
-                emoji = emojis[idx]
-                daos[emoji] = guild.id
-                embed.add_field(name=guild.name, value=emoji)
-            message = await ctx.followup.send(embed=embed)
-            for emoji in emojis:
-                await message.add_reaction(emoji)
-            await Redis.set(
-                ctx.author.id,
-                build_cache_value(
-                    ThreadKeys.UPDATE_PROFILE.value,
-                    UpdateProfile(
-                        ctx.author.id,
-                        hashlib.sha256("".encode()).hexdigest(),
-                        message.id,
-                        "",
-                    ).steps.hash_,
-                    "",
-                    message.id,
-                    metadata={"daos": daos},
-                ),
-            )
+
+# if bool(strtobool(constants.Bot.is_dev)):
+# @keating No current development functions
+# wasn't sure how to make sure this didn't end up grabbng the below functions
```

Comment:
This can be removed. We can always add back if we want to gate slash commands in the guture

Reply:
ok cool sounds good.  I'll make the changes and send